Stats: Display placeholders, defaults and help

wagner-certat opened the pull request.

bernhardreiter commented:
Hi @wagner-certat, thanks for the pull request!

* Because I've looked at merged #69 first with a fix, this one does not fully merge anymore 
as `allowedSubs` has been changed. I did not see that you were using this here. Sorry about this.

* I've also noticed that https://github.com/Intevation/intelmq-fody-backend/pull/19/files does not contain default values. Where does the `default` value come from?

* `removeFilter()` is missing?
* The '?' is a button, but the click does not do anything, a tooltip possibly should be over the input field itself?

wagner-certat commented:
>     * Because I've looked at merged #69 first with a fix, this one does not fully merge anymore
>       as `allowedSubs` has been changed. I did not see that you were using this here. Sorry about this.
>       (for testing you could change `allowedSubs` back to a dict and do the sorting in the v-for loop directly).

I did the merges and adapted the code as bf22e6b83482bdfcaceb374788be029e94f21bf7 changed the allowedSubs from a handy object with the itms to a list of lists with the actual items as object.

>     * I've also noticed that https://github.com/Intevation/intelmq-fody-backend/pull/19/files does not contain default values. Where does the `default` value come from?

It's optional of course. I don't have a good universal example at hand for it's usage though.

>     * `removeFilter()` is missing?

That's a change upstream, I removed change this now - it's no longer necessary.

>     * The '?' is a button, but the click does not do anything, a tooltip possibly should be over the input field itself?

That was the easiest for me to implement and I also think it is very obvious and intuitive for the users when they want to have some help. I removed this now and used the input field itself, which does not work. If you have an idea how to show this, please let me know.

bernhardreiter commented:
Showing the help text should be possible by using a popup when hovering over the field, I'll have to look it up how to do it, though. (And we currently do not have much contracted time to work on this.)
